recipe-client-addon/test/browser/browser_PreferenceExperiments.js

```diff
@@ -214,9 +215,9 @@ add_task(withMockPreferences(async function(mockPreferences) {
 
   PreferenceExperiments.stopAllObservers();
   stop.restore();
-}));
+})));
 
-add_task(async function testHasObserver() {
+add_task(withMockExperiments(async function testHasObserver() {
```

Comment:
I added these mocks because they make sure that observers get properly cleaned up. We don't strictly need them on every test, but they make the tests more consistent.